Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace КомароваЗадание5._2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string str = "";
+            str = "Перемена";
```

Comment:
В задании исходное слово начинается с маленькой буквы.
```suggestion
            str = "перемена";
```